content/100-getting-started/02-setup-prisma/200-add-to-existing-project/110-relational-databases/100-connect-your-database.mdx

```diff
@@ -208,6 +208,64 @@ Adjust the connection URL to match your setup - see [Microsoft SQL Server connec
 
 </SwitchTech>
 
+<SwitchTech technologies={['*', 'cockroachdb']}>
+
+```prisma file=prisma/schema.prisma
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+```
+
+Note that the default schema created by `prisma init` uses PostgreSQL as the `provider`. For PlanetScale, you need to edit the `datasource` block to use the `cockroachdb` provider instead:
+
+```prisma file=prisma/schema.prisma highlight=2;edit
+datasource db {
+  provider = "cockroachdb"
+  url      = env("DATABASE_URL")
+}
+```
+
+You will also need to add the `cockroachdb` preview feature flag to the `generator` block of your `schema.prisma` file:
+
+```prisma file=schema.prisma highlight=3;add
+generator client {
+  provider        = "prisma-client-js"
+  previewFeatures = ["cockroachdb"]
+}
+
+datasource db {
+  provider = "cockroachdb"
+  url      = env("DATABASE_URL")
+}
+```
+
+The `url` is [set via an environment variable](/guides/development-environment/environment-variables) which is defined in `.env`. For a CockroachDB database, the [connection URL](/reference/database-reference/connection-urls) looks similar to this:
+
+```bash file=.env
+DATABASE_URL="postgresql://myusername:mypassword@free-tier5.gcp-europe-west1.cockroachlabs.cloud:26257/defaultdb?sslmode=verify-full&sslrootcert=$HOME/.postgresql/root.crt&options=--mycluster
+```
+
+You now need to adjust the connection URL to point to your own database.
+
+The format of the connection URL for your database depends on the database you use. CockroachDB uses the PostgreSQL connection URL format, which has the following structure (the parts spelled all-uppercased are _placeholders_ for your specific connection details):
+
+```no-lines
+postgresql://USER:PASSWORD@HOST:PORT/DATABASE?sslmode=verify-full&sslrootcert=$HOME/.postgresql/root.crt&options=--CLUSTER
+```
+
+Here's a short explanation of each component:
+
+- `USER`: The name of your database user
+- `PASSWORD`: The password for your database user
+- `PORT`: The port where your database server is running. The default for CockroachDB is `26257`.
+- `DATABASE`: The name of the database
+- `CLUSTER`: The name of the database cluster your database is running on
+
+To find your connection string, click the 'Connect' button on the overview page for your database cluster, and select the 'Connection string' tab.
```

Comment:
For local CockroachDB installations it is shown in command line (see https://www.cockroachlabs.com/docs/v21.2/hello-world-node-postgres?filters=local)